dolphinscheduler-dao/src/main/resources/sql/dolphinscheduler_mysql.sql

```diff
@@ -665,7 +665,7 @@ CREATE TABLE `t_ds_queue` (
 -- ----------------------------
 -- Records of t_ds_queue
 -- ----------------------------
-INSERT INTO `t_ds_queue` VALUES ('1', 'default', 'default', null, null);
+INSERT IGNORE INTO `t_ds_queue` VALUES ('1', 'default', 'default', null, null);
```

Comment:
Does postgres need to be modified?

Reply:
It seems PostgreSQL doesn't support insert ignore

Reply:
It seems that only versions above 9.5 are supported...
`https://stackoverflow.com/questions/1009584/how-to-emulate-insert-ignore-and-on-duplicate-key-update-sql-merge-with-po`

Reply:
Yes, so I don't want to modify the postgresql.